### PR TITLE
adding mission, vision and values post

### PR DESCRIPTION
**Pull request by julien51**
# Description

All in the title ;)

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: unlock-protocol.com/blog/mission-vision.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+subTitle: We empower creators to capture the value they create for their community, on their own terms
+authorName: Julien Genestoux
+publishDate: June 27, 2019
+description: Identifying Mission, Vision and Values informs business strategy and paints a holistic and accurate picture of a business.
+image: /static/images/blog/whatisunlock.jpg
+---
+
+A recipe to go from zero to one cannot exist. Creating a company, a team, a product or a protocol require a lot of work and faith. Our goal is ambitious, but it can only be achieved an incredible amount of small steps (4000 [Pull Requests](https://github.com/unlock-protocol/unlock/pulls) as of Tuesday!).
+
+This week, we took some time to think about our **mission**, which is what we do, our **vision**, which is why we do it and our **values**, which describe how we do it. We got some help and it took us a full day, but this work will help us in each decision and will act as a "scaffold" for the whole team.
```
Comment (benwerd) [inline unlock-protocol.com/blog/mission-vision.md]:
```suggestion
This week, we took some time to think about our **mission**, which is what we do, our **vision**, which is why we do it, and our **values**, which describe how we do it. We got some help and it took us a full day, but this work will help us in each decision and will act as a "scaffold" for the whole team.
```

File: unlock-protocol.com/blog/mission-vision.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Our Vision
+
+We believe that, now, most creators are not able to capture enough of the value that they create. Open source developers, musicians, writters, or even curators provide an incredible value for all users of the internet and beyond, but we also have to acknoledge that most of them are not being rewarded for that work, and when they are, it is often through gatekeepers, middlemen who set the terms and too often end up competing with them.
```
Comment (benwerd) [inline unlock-protocol.com/blog/mission-vision.md]:
```suggestion
We believe that, now, most creators are not able to capture enough of the value that they create. Open source developers, musicians, writers, or even curators provide incredible value for all users of the internet and beyond. But we also have to acknowledge that most of them are not being rewarded for that work, and when they are, it is often through gatekeepers: middlemen who set the terms and too often end up competing with them.
```

File: unlock-protocol.com/blog/mission-vision.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+We know there is a better way and this is our vision:
```
Comment (benwerd) [inline unlock-protocol.com/blog/mission-vision.md]:
```suggestion
We know there is a better way. This is our vision:
```

File: unlock-protocol.com/blog/mission-vision.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Values define "how" we want to achieve our mission and realize the vision. They provide principles that we want to live by, both internally, but also externally, with users of the protocol and members of our own community. Here are the ones we chose:
+
+- Collaborative
+- Human Centered
+- Mission Driven
+- Simple
+- Transparent
+
+Of course, in that list there are things that we already apply, and others which are, at least until now, more aspiratonial.
```
Comment (benwerd) [inline unlock-protocol.com/blog/mission-vision.md]:
```suggestion
Of course, in that list there are things that we already apply, and others which are, at least for now, more aspirational.
```

File: unlock-protocol.com/blog/mission-vision.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Based on that list, it should be pretty clear that your input matters to us and we hope that you'll provide us guidance on how to adopt these principles during our journey! Please, reach out to us, in comments below, on [Telegram](https://t.me/unlockprotocol), or even privately using the Intercom widget in the lower right corner of this page.
```
Comment (benwerd) [inline unlock-protocol.com/blog/mission-vision.md]:
```suggestion
Based on that list, it should be pretty clear that your input matters to us. We hope that you'll provide us guidance on how to adopt these principles during our journey! Please, reach out to us, in comments below, on [Telegram](https://t.me/unlockprotocol), or even privately using the Intercom widget in the lower right corner of this page.
```

**Pull request merged**